microsoft-power-point/microsoft-power-point-quiz.md

```diff
@@ -1,156 +1,148 @@
 ## Microsoft Power Point Assessment
 
-#### Q1. When you create a new, blank presentation. you begin with one blank slide. What is the layout name of this slide?
+#### Q1. When you create a new, blank presentation, you begin with one blank slide. What is the layout name of this slide?
 
-    Blank
-    = Title slide
-    Title and Content
-    Title Onty
+- [ ] Blank
+- [x] Title slide
+- [ ] Title and Content
+- [ ] Title Onty
 
-#### Q2. when do you use to align objects on a slide to one another?
+#### Q2. What do you use to align objects on a slide to one another?
 
-    = Layout tools
-    Arrange rools
-    Design Ideas feature
-    Picture tools
+- [ ] Layout tools
+- [x] Arrange tools
+- [ ] Design Ideas feature
+- [ ] Picture tools
 
 #### Q3. Which option must be enabled if you want to manipulate the playback of a video file during a presentation?
 
-    Record Side Show
-    Use Presenter View
-    Show Playback Controls
-    = Show Media Controls
+- [ ] Record Side Show
+- [ ] Use Presenter View
+- [ ] Show Playback Controls
+- [x] Show Media Controls
 
 #### Q4. Which feature analyzes the content of your slides and offers suggested options?
 
-    = Design ideas
-    Smartart
-    Browse for Themes
-    Add-Ins
+- [x] Design ideas
+- [ ] Smartart
+- [ ] Browse for Themes
+- [ ] Add-Ins
 
 #### Q5. Where can you configure advanced PDF options?
 
-    = the Options tab
-    the Home tab
-    the Print dialog box
-    the Share tab
+- [ ] the Options tab
+- [ ] the Home tab
+- [x] the Print dialog box
+- [ ] the Share tab
 
 #### Q6. Which tab is best for adding various types of objects to slides?
 
-    Animations
-    view
-    = File
-    insert
+- [ ] Animations
+- [ ] view
+- [ ] File
+- [x] Insert
 
 #### Q7. Which SmartArt category can you use to make informational graphics out of bullet points?
 
-    = all of these answers
-    Pyramid
-    List
-    Matrix
+- [x] all of these answers
+- [ ] Pyramid
+- [ ] List
+- [ ] Matrix
 
 #### Q8. What is NOT checked when you run the Accessibility Checker?
 
-    reading order
-    = slide titles
-    missing alt text
-    grammar
+- [ ] reading order
+- [ ] slide titles
+- [ ] missing alt text
+- [x] grammar
```

Comment:
- [x] grammar // [Source](https://support.microsoft.com/en-us/office/rules-for-the-accessibility-checker-651e08f2-0fc3-4e10-aaca-74b4a67101c1?ns=powerpnt&version=16&ui=en-us&rs=en-us&ad=us)